exo-go/src/app_runner/index.go

```diff
@@ -12,30 +12,31 @@ import (
 	"github.com/Originate/exosphere/exo-go/src/logger"
 	"github.com/Originate/exosphere/exo-go/src/process_helpers"
 	"github.com/Originate/exosphere/exo-go/src/service_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/service_restarter"
 	"github.com/Originate/exosphere/exo-go/src/types"
 	"github.com/fatih/color"
 	"github.com/pkg/errors"
 )
 
 // AppRunner runs the overall application
 type AppRunner struct {
-	AppConfig            types.AppConfig
-	Logger               *logger.Logger
-	AppDir               string
-	homeDir              string
-	Env                  map[string]string
-	DockerConfigLocation string
+	AppConfig        types.AppConfig
+	Logger           *logger.Logger
+	AppDir           string
+	homeDir          string
+	Env              map[string]string
+	DockerComposeDir string
```

Comment:
In a separate PR please make this a computed property instead of saving it since it is related to appDir